<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseModulePrototypeAnimationAddition</class>
 <widget class="QWidget" name="DatabaseModulePrototypeAnimationAddition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,1,0" columnstretch="1,0" rowminimumheight="0,0,0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="database::ui::module::PrototypeBase" name="m_pModuleBase" native="true"/>
   </item>
   <item row="0" column="1" rowspan="4">
    <widget class="PrototypeTooltipList" name="m_pModuleSpriteList"/>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="m_pTypeBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QSlider" name="m_pZoom">
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="sliderPosition">
      <number>100</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <widget class="QGraphicsView" name="m_pView">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="dragMode">
      <enum>QGraphicsView::ScrollHandDrag</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="animation::ui::module::SelectedSprite" name="m_pModuleSelected" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>database::ui::module::PrototypeBase</class>
   <extends>QWidget</extends>
   <header>databasemoduleprototypebase.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>animation::ui::module::SelectedSprite</class>
   <extends>QWidget</extends>
   <header>animationmoduleselectedsprite.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PrototypeTooltipList</class>
   <extends>QListView</extends>
   <header>prototypetooltiplist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
